integrations/repo_test.go

```diff
@@ -149,16 +149,16 @@ func TestViewRepoWithSymlinks(t *testing.T) {
 	htmlDoc := NewHTMLParser(t, resp.Body)
 	files := htmlDoc.doc.Find("#repo-files-table > TBODY > TR > TD.name > SPAN")
 	items := files.Map(func(i int, s *goquery.Selection) string {
-		cls, _ := s.Find("SPAN").Attr("class")
+		cls, _ := s.Find("SVG").Attr("class")
 		file := strings.Trim(s.Find("A").Text(), " \t\n")
 		return fmt.Sprintf("%s: %s", file, cls)
 	})
 	assert.Equal(t, len(items), 5)
-	assert.Equal(t, items[0], "a: octicon octicon-file-directory")
```

Comment:
Could you check the hash content of attr href of use element to keep the test goal ?

Reply:
Currently trying to do this, but `goquery` doesn't want to find `use` nodes for some reason...

Reply:
Instead, I added the icon as a class to the `svg` element. This would also give us the option to style each icon individually (if for some reason we decided that was a good idea) and makes them easier to select in JS.